package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jest-styled-components",
-  "version": "5.0.1",
+  "version": "5.1.0",
```

Comment:
The next version is going to be 6, as it will include other PRs (and breaking changes), but please revert this for now - I will bump the version once merged.

Reply:
Done.